<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="dataTab">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <attribute name="title">
        <string>Данные</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="5" column="0">
         <widget class="QTableView" name="subject_table"/>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="cabinetsLabel">
          <property name="text">
           <string>Кабинеты</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QTableView" name="group_table"/>
        </item>
        <item row="5" column="2">
         <widget class="QTableView" name="cabinets_table"/>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="timeLabel">
          <property name="text">
           <string>Время</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="groupsLabel">
          <property name="text">
           <string>Группы</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="fieldsLabel">
          <property name="text">
           <string>Предметы</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QTableView" name="time_table"/>
        </item>
        <item row="6" column="3">
         <layout class="QGridLayout" name="gridLayout_time_btn"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="scheduleTab">
       <attribute name="title">
        <string>Расписание</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="openFile">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
